models/student.model.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt')
+
+const studentSchema = mongoose.Schema({
+  email: {
+    type: String,
+    match: /.+\@.+\..+/,
+    required: [true, "El email es requerido"],
+    validate: {
+      validator: async function(value) {
+        const student = await Student.findOne({ email: value })
```

Comment:
What happen is Student and tutor have the same email, can they be duplicated?

Reply:
There can not be a student and a tutor with the same email, I added an extra validation